.circleci/config.yml

```diff
@@ -32,13 +32,10 @@ install_gruntwork_utils: &install_gruntwork_utils
     gruntwork-install --module-name "kubernetes-circleci-helpers" --repo "https://github.com/gruntwork-io/module-ci" --tag "${MODULE_CI_VERSION}"
     gruntwork-install --binary-name "terratest_log_parser" --repo "https://github.com/gruntwork-io/terratest" --tag "${TERRATEST_LOG_PARSER_VERSION}"
     configure-environment-for-gruntwork-module \
-      --circle-ci-2-machine-executor \
       --terraform-version ${TERRAFORM_VERSION} \
       --terragrunt-version ${TERRAGRUNT_VERSION} \
       --packer-version ${PACKER_VERSION} \
-      --use-go-dep \
       --go-version ${GOLANG_VERSION} \
```

Comment:
I intentionally removed the `--go-src-path`. Contrary to expectations, `go get -v -t -d` does not honor the `go.sum`, which causes it to pull in the latest source code for `client-go`. Naturally, that breaks `terratest`.

I need to figure out what is the correct way to pull in dependencies. So far, `go mod tidy` is the most reliable way.